telemetry/definitions/commonDefinitions.json

```diff
@@ -1284,6 +1314,21 @@
             "type": "string",
             "description": "Application framework being used"
         },
+        {
+            "name": "generatedCharactersCount",
+            "type": "int",
+            "description": "Number of characters of code generated"
+        },
+        {
+            "name": "generatedCount",
```

Comment:
this name is unhelpful.  why not `generatedTestCasesCount`?

Reply:
Please refer this [PR](https://github.com/aws/aws-toolkit-common/pull/905#discussion_r1837264303)
Long story short Toolkit want to reuse the fields so the ask is to keep this as generic as possible.

Reply:
Fields already have the context of the metrics they are attached to. There is no need to name each field in a globally-unique way, and it's actually counterproductive.

Reply:
You probably don't need this field, just use the `count` field: https://github.com/aws/aws-toolkit-common/blob/fac1daaf6822a666fbcd07cd1a11c77734e0cf6b/telemetry/definitions/commonDefinitions.json#L937